.github/workflows/code-coverage.yml

```diff
@@ -0,0 +1,62 @@
+name: Code Coverage
+
+on:
+  push:
+    branches: [ 'master' ]
+  pull_request:
+    branches: [ 'master' ]
+
+jobs:
+  coverage:
+      name: Code Coverage
+
+      runs-on: macos-latest
+      env:
+        BUILD_WRAPPER_OUT_DIR: buildwrapper # Directory where build-wrapper output will be placed
+
+      steps:
+        - name: Checkout
+          uses: actions/checkout@v2
+          with:
+            fetch-depth: 0
+
+        - name: Run xcodebuild with tests
+          run: xcodebuild -project OmiseSDK.xcodeproj/ -scheme OmiseSDK -derivedDataPath Build/ -destination 'platform=iOS Simulator,name=iPhone 11,OS=16.2' -enableCodeCoverage YES clean build test CODE_SIGN_IDENTITY="" CODE_SIGNING_REQUIRED=NO
```

Comment:
Should we remove the existing step that runs tests? https://github.com/omise/omise-ios/blob/312ff06f9bae9e2cc4ae1688883071e6a9cbc0b0/.github/workflows/main.yml#L13

Reply:
We moved everything in one action and removed old testing step